cmd/dp-deployer/main.go

```diff
@@ -127,6 +144,16 @@ func startHealthChecks(ctx context.Context, cfg *config.Configuration, vaultChec
 		log.Event(ctx, "error adding check for vault", log.ERROR, log.Error(err))
 	}
 
+	if err := hc.AddCheck("S3 secret", s3sChecker.Checker); err != nil {
+		hasErrors = true
+		log.Event(ctx, "error adding check for S3 secrets", log.ERROR, log.Error(err))
+	}
+
+	if err := hc.AddCheck("S3 deployment", s3dChecker.Checker); err != nil {
+		hasErrors = true
+		log.Event(ctx, "error adding check for S3 deployments", log.ERROR, log.Error(err))
+	}
+
```

Comment:
👇 I realise this isn't part of this review but I think its good practice to ensure the only place the app does an `os.Exit` is in the `main func`. So I would consider amending the return type of this function to
```
(*healthcheck.HealthCheck, bool)
```
and then `return hc, hasErrors` and do
```
if hasErrors {
    os.Exit(1)
}
```
in main.